drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -0,0 +1,70 @@
+# Kconfig - STM32 RTC configuration options
+#
+# Copyright (c) 2018 Workaround GmbH
+# Copyright (c) 2018 Allterco Robotics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config RTC_STM32
+	bool "STM32 RTC Driver"
+	depends on SOC_FAMILY_STM32 && SOC_SERIES_STM32L4X
```

Comment:
Remove dependency on L4. It is sufficient to control activation per series (in arch)